plugins/evernote.koplugin/main.lua

```diff
@@ -122,7 +130,7 @@ function EvernoteExporter:addToMainMenu(menu_items)
                                 self:login()
                             end
                         }
-                    } or nil
+                                                       } or nil
```

Comment:
Did you use some automated method for this? There are a few really weird changes here, like this line and on lines further down below. I really just meant in the previous diff.